### PR TITLE
typo: Double word "the"

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/ApiManagement/ApiManagement/help/Update-AzApiManagementRegion.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ PS C:\>$apimService = Update-AzApiManagementRegion -ApiManagement $apimService -
 PS C:\>$updatedService = Set-AzApiManagement -InputObject $apimService -PassThru
 ```
 
-This command gets the API Management Premium SKU service, having regions in South Central US and North Central US. It then increases the Capacity of the North Central US region to 2 using the **Update-AzApiManagementRegion**. The next cmdlet Set-AzApiManagement applies the configuration change to the the Api Management service.
+This command gets the API Management Premium SKU service, having regions in South Central US and North Central US. It then increases the Capacity of the North Central US region to 2 using the **Update-AzApiManagementRegion**. The next cmdlet Set-AzApiManagement applies the configuration change to the Api Management service.
```
Comment (markcowl) [inline src/ApiManagement/ApiManagement/help/Update-AzApiManagementRegion.md]: This needs an update to the changelog for ApiManagement, per the release checklist

File: src/Compute/Compute/help/New-AzVmss.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ New-AzVmss [[-ResourceGroupName] <String>] [-VMScaleSetName] <String> [-AsJob] [
 
 ## DESCRIPTION
 The **New-AzVmss** cmdlet creates a Virtual Machine Scale Set (VMSS) in Azure.
-Use the simple parameter set (`SimpleParameterSet`) to quickly create a pre-set VMSS and associated resources. Use the default parameter set (`DefaultParameter`) for more advanced scenarios when you need to precisely configure each component of the the VMSS and each associated resource before creation.
+Use the simple parameter set (`SimpleParameterSet`) to quickly create a pre-set VMSS and associated resources. Use the default parameter set (`DefaultParameter`) for more advanced scenarios when you need to precisely configure each component of the VMSS and each associated resource before creation.
```
Comment (markcowl) [inline src/Compute/Compute/help/New-AzVmss.md]: Needs a corresponding changelog change

File: src/DataLakeStore/DataLakeStore/help/Get-AzDataLakeStoreItemPermission.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Get-AzDataLakeStoreItemPermission [-Account] <String> [-Path] <DataLakeStorePath
 ```
 
 ## DESCRIPTION
-The **Get-AzDataLakeStoreItemPermission** cmdlet gets the the permission octal of a file or folder in Data Lake Store.
+The **Get-AzDataLakeStoreItemPermission** cmdlet gets the permission octal of a file or folder in Data Lake Store.
```
Comment (markcowl) [inline src/DataLakeStore/DataLakeStore/help/Get-AzDataLakeStoreItemPermission.md]: Needs a corresponding changelog change

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesAsrProtectionContainerMapping.md [hunk 4]
```diff
@@ -91,7 +91,7 @@ Accept wildcard characters: False
 ```
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesAsrProtectionContainerMapping.md]: Do not change the json file

File: src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesAsrProtectionContainerMapping.md [hunk 5]
```diff
@@ -91,7 +91,7 @@ Accept wildcard characters: False
...
 
 ### -ProtectionContainer
-Get protection container mappings corresponding to the the specified ASR protection container object.
+Get protection container mappings corresponding to the specified ASR protection container object.
```
Comment (markcowl) [inline src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesAsrProtectionContainerMapping.md]: Needs a corresponding changelog change

File: src/Sql/Sql/Elastic Pools/Model/AzureSqlElasticPoolModel.cs [hunk 6]
```diff
@@ -102,7 +102,7 @@ public class AzureSqlElasticPoolModel
         public double? DatabaseCapacityMax { get; set; }
 
         /// <summary>
-        /// Gets or sets the the sku family of the elastic pool
+        /// Gets or sets the sku family of the elastic pool
```
Comment (markcowl) [inline src/Sql/Sql/Elastic Pools/Model/AzureSqlElasticPoolModel.cs]: Needs a corresponding changelog change

File: src/Sql/Sql/help/Az.Sql.md [hunk 7]
```diff
@@ -635,6 +635,6 @@ Updates an Azure SQL Database Sync Member.
 
 ### [Update-AzSqlSyncSchema](Update-AzSqlSyncSchema.md)
 Update the sync schema for a sync member database or a sync hub database.
-It will get the the latest database schema from the real database and then use it refresh the schema cached by Sync metadata database.
+It will get the latest database schema from the real database and then use it refresh the schema cached by Sync metadata database.
```
Comment (markcowl) [inline src/Sql/Sql/help/Az.Sql.md]: Needs a corresponding changelog change

File: src/Sql/Sql/help/Update-AzSqlSyncSchema.md [hunk 8]
```diff
@@ -9,7 +9,7 @@ schema: 2.0.0
 
 ## SYNOPSIS
 Update the sync schema for a sync member database or a sync hub database.
-It will get the the latest database schema from the real database and then use it refresh the schema cached by Sync metadata database.
+It will get the latest database schema from the real database and then use it refresh the schema cached by Sync metadata database.
```
Comment (markcowl) [inline src/Sql/Sql/help/Update-AzSqlSyncSchema.md]: Needs a corresponding changelog change

**Comment by markcowl** [comment]
@nschonni We really appreciate all of these PRs, however, this PR does not meet the PR guidelines for this repo

- All PRs must contain a description
- All PRs must use the PR template, which is automatically added when you create a PR through Github UI.
- All changes to cmdlet code or help must have an accompanying change to the relevant module changelog

**Comment by nschonni** [comment]
> * All PRs must contain a description
> 
> * All PRs must use the PR template, which is automatically added when you create a PR through Github UI.

Yeah, I use the PR extension in VS Code that seems to skip that

>     * All changes to cmdlet code or help must have an accompanying change to the relevant module changelog

I'll take a look at that.

Is it better to combine these PRs, or is separate preferred?

**Comment by markcowl** [comment]
@nschonni It is easier to have a smaller number, so CI doesn't run on each PR, and this also makes it easier to track changelog entries.

For PRs this simple, you cna skip the checklist, but the changelog change is required for us to track the modules the need to be re-released and have documentation updated.

**Comment by nschonni** [comment]
OK, I'm closing the ones that you didn't trigger the CI on already, do submit one large PR

**Pull request closed without merging**